src/scene/render-pass.js

```diff
@@ -0,0 +1,21 @@
+/** @typedef {import('../graphics/render-target.js').RenderTarget} RenderTarget */
+
+/**
+ * A render pass represents a node in the frame graph, and encapsulates a system which
+ * renders to a render target using an execution callback.
+ *
+ * @ignore
+ */
+class RenderPass {
+    /** @type {string} */
+    name;
+
+    constructor(renderTarget, execute) {
```

Comment:
Might be a good idea to add JSDoc comments for this constructor. `execute` in particular is unexplained here.

Reply:
yep, done